packages/accounts/src/account_2fa.ts

```diff
@@ -249,6 +293,10 @@ export class Account2FA extends AccountMultisig {
         };
     }
 
+    /**
+     * Gets the 2FA method (kind and detail).
+     * @returns {Promise<{ kind: string, detail: string } | null>} - A promise that resolves to the 2FA method.
```

Comment:
(no action required) Why is `| null` added only here and not in other functions?

Reply:
mb, will remove it